Spinner/Spinner.js

```diff
@@ -55,7 +56,7 @@ const SpinnerCore = kind({
 	},
 
 	defaultProps: {
-		type: 'searching'
+		type: $L('searching')
```

Comment:
This looks weird. This is a _public_ prop and the value should be a plane text. This value is used just to test like `type === 'searching'.

Reply:
I agree with your opinion. Fixed to compare type with plane text.